c10/core/DispatchKey.h

```diff
@@ -66,6 +66,7 @@ enum class DispatchKey : uint8_t {
   Vulkan,
   Metal,
   XPU, // For out of tree Intel's heterogeneous computing plug-in
+  HPU, // HPU / Habana
```

Comment:
Can you also mention "out of tree & closed source integration" in the comment here?

Reply:
Added